account/keystore_nimbus.go

```diff
@@ -0,0 +1,49 @@
+// +build nimbus
+
+package account
+
+import (
+	"io/ioutil"
+	"os"
+
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+
+	gethbridge "github.com/status-im/status-go/eth-node/bridge/geth"
+	// nimbusbridge "github.com/status-im/status-go/eth-node/bridge/nimbus"
+	"github.com/status-im/status-go/eth-node/types"
+)
+
+// makeAccountManager creates ethereum accounts.Manager with single disk backend and lightweight kdf.
+// If keydir is empty new temporary directory with go-ethereum-keystore will be intialized.
+func makeAccountManager(keydir string) (manager *accounts.Manager, err error) {
```

Comment:
We're doing a LOT of code duplication with stuff like this. The only difference I see here is the name of the temporary directory which is `nimbus-keystore`. Do we really have to duplicate all this code just to change that?

I guess the main difference is the `gethbridge` used. If that's the case then why can't we parametrize it to accept an interface that can accept both types of bridges?